sklearn/datasets/lfw.py

```diff
@@ -72,36 +89,37 @@ def check_fetch_lfw(data_home=None, funneled=True, download_if_missing=True):
     lfw_home = join(data_home, "lfw_home")
 
     if funneled:
-        archive_path = join(lfw_home, FUNNELED_ARCHIVE_NAME)
         data_folder_path = join(lfw_home, "lfw_funneled")
-        archive_url = BASE_URL + FUNNELED_ARCHIVE_NAME
+        archive_path = join(lfw_home, FUNNELED_ARCHIVE.filename)
```

Comment:
we can't do it like this 'cos its downloading the files in different directories. I don't really know why we are doing it like this 'cos two lines afterwards we delete them. But the test require it. But i agree that it can be simplified compare with how it was.